website/events/emails.py

```diff
@@ -56,5 +56,8 @@ def notify_organiser(event, registration):
     EmailMessage(
         subject,
         text_message,
-        to=[event.organiser.contact_mailinglist.name + "@thalia.nu"],
+        to=[
+            organiser.contact_mailinglist.name + "@thalia.nu"
```

Comment:
I think we shouldn't hardcode thalia.nu, I think we can use the default site from the sites framework

Reply:
Or the base url from the settings. I think we use that in most places.

Reply:
Added it! :D